### PR TITLE
add annotations documentation.

### PR DESCRIPTION
**Pull request by stuartnelson3**
@juliusv

### CODE REVIEW

File: content/docs/visualization/promdash.md [hunk 1]
```diff
@@ -230,10 +230,63 @@ show `serverID` fields, and never `serverURL`, since any uploaded JSON
...
+render them on the graph. If you have a server running that stores events with
+a timestamp, you can query it to render annotations on your widgets.
+
+[![Annotation screenshot](/assets/annotations_example.png)](/assets/annotations_example.png)
+
+To enable, first select "Display annotation tags" via the global dashboard
+setting. Each input can hold an arbitrary number of comma-separated tags that
+will be used to query your designated annotations server. Once you have added
+your tags you can hide the annotation list by un-checking annotations setting
+in the global dashboard setting.
```
Comment (juliusv) [inline content/docs/visualization/promdash.md]: s/setting/settings/

Comment (juliusv) [inline content/docs/visualization/promdash.md]: s/via/via the/

**Comment by juliusv** [comment]
@stuartnelson3 PTAL

File: content/docs/visualization/promdash.md [hunk 2]
```diff
@@ -188,6 +150,112 @@ will then automatically resize the Graphite graph to fit its widget by
...
+The value of the variable value may then be inserted into relevant text input
+fields in a widget's settings.
+
+A variable is interpolated into a text input field by enclosing it with a set
+of two curly braces: `{{variable-name}}`. The text inputs that support template
+variable interpolation are:
+
+* The widget title.
+* The expression input fields in a Prometheus graph.
+* The URL input of an iframe widget.
```
Comment (stuartnelson3) [inline content/docs/visualization/promdash.md]: annotation tags accept them, too

File: content/docs/visualization/promdash.md [hunk 3]
```diff
@@ -63,10 +63,10 @@ have an effect on the dashboard as a whole:
     data from Prometheus.
   * **Show annotation tags**: Select whether to show the UI for editing tags
     that speficy which annotations to load and display in graphs. See the
```
Comment (stuartnelson3) [inline content/docs/visualization/promdash.md]: speficy => specify

**Comment by stuartnelson3** [comment]
other than my two comments, :+1:

**Comment by juliusv** [comment]
Fixed both of these - thanks!

**Pull request merged**